File: tests/tests/fixtures/python_repo/repositories/payment_repository.py
"""repositories/payment_repository.py — fixture for DocWiki tests."""

from __future__ import annotations


class PaymentRepository:
    """Persists payment records."""

    def save(self, order) -> str:
        return str(id(order))

    def find(self, order_id: str) -> str:
        return order_id

File: tests/tests/fixtures/java_repo/src/main/java/com/example/repository/PaymentRepository.java
package com.example.repository;

public class PaymentRepository {
    public String save(Object order) {
        return String.valueOf(order.hashCode());
    }

    public String find(String orderId) {
        return orderId;
    }

    // Overloaded method
    public String find(String orderId, boolean includeArchived) {
        return orderId + ":" + includeArchived;
    }
}
